sdk/servicebus/service-bus/src/serviceBusAtomManagementClient.ts

```diff
@@ -293,13 +295,30 @@ export class ServiceBusManagementClient extends ServiceClient {
     operationOptions?: OperationOptions
   ): Promise<NamespacePropertiesResponse> {
     log.httpAtomXml(`Performing management operation - getNamespaceProperties()`);
-    const response: HttpOperationResponse = await this.getResource(
-      "$namespaceinfo",
-      this.namespaceResourceSerializer,
-      operationOptions
+    const { span, spanOptions } = createSpan(
+      "ServiceBusManagementClient-getNamespaceProperties",
+      operationOptions?.tracingOptions
     );
+    try {
+      const response: HttpOperationResponse = await this.getResource(
+        "$namespaceinfo",
+        this.namespaceResourceSerializer,
+        {
+          ...operationOptions,
+          tracingOptions: { ...operationOptions?.tracingOptions, spanOptions }
```

Comment:
in other clients we had the `createSpan` helper pass back an updated `operationOptions` to avoid having to re-assemble the new options with the updated spanOptions. It feels like that might be useful here too?

Reply:
I think I should refactor.

Reply:
Done